catalog/tests/dags/common/test_ingestion_server.py

```diff
@@ -1,11 +1,49 @@
-from unittest.mock import MagicMock
+from datetime import timedelta
+from unittest import mock
 
 import pytest
-from airflow.exceptions import AirflowSkipException
+import requests
+from airflow.exceptions import AirflowException, AirflowSkipException
+from airflow.models import DagRun, TaskInstance
+from airflow.models.dag import DAG
+from airflow.utils.session import create_session
+from airflow.utils.state import DagRunState, TaskInstanceState
+from airflow.utils.timezone import datetime
+from airflow.utils.types import DagRunType
 
 from common import ingestion_server
 
 
+TEST_START_DATE = datetime(2022, 2, 1, 0, 0, 0)
+TEST_DAG_ID = "api_healthcheck_test_dag"
+
+
+@pytest.fixture(autouse=True)
+def clean_db():
+    with create_session() as session:
+        # synchronize_session='fetch' required here to refresh models
+        # https://stackoverflow.com/a/51222378 CC BY-SA 4.0
+        session.query(DagRun).filter(DagRun.dag_id.startswith(TEST_DAG_ID)).delete(
+            synchronize_session="fetch"
+        )
+        session.query(TaskInstance).filter(
+            TaskInstance.dag_id.startswith(TEST_DAG_ID)
+        ).delete(synchronize_session="fetch")
+
+
+@pytest.fixture()
+def healthcheck_dag():
+    index_suffix = "my-test-suffix"
+
+    # Create a DAG that just has an api_health_check task
+    with DAG(dag_id=TEST_DAG_ID, schedule=None, start_date=TEST_START_DATE) as dag:
+        ingestion_server.api_health_check(
+            media_type="image", index_suffix=index_suffix, timeout=timedelta(days=1)
+        )
```

Comment:
Nit: it seems like `index_suffix` can just be defined in-line here at the `api_health_check` call